jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -63,7 +63,7 @@
     # being deployed as a CI soak job to periodically bring up
     # and tear down the clusters for the federation pull job.
     - kubernetes-pull-gce-federation-deploy:
-        blocker: pull-kubernetes-federation-e2e-gce
```

Comment:
a few lines above - do you still need the old soak job?

Reply:
i would revisit that and adapt to new federation repo.

Reply:
removing the federation soak jobs for now as they have been broken for a while. we shall ass new soak jobs for federation after federation code is moved out of core.